features/command_line/only_failures.feature

```diff
@@ -103,6 +104,11 @@ Feature: Only Failures
     When I run `rspec --next-failure`
     Then the output should contain "All examples were filtered out"
 
+  Scenario: Running `rspec --only-failures` with spec files that pass doesn't run anything
+    When I run `rspec spec/passing_spec.rb --only-failures`
+    Then the output from "rspec spec/passing_spec.rb --only-failures" should contain "0 examples, 0 failures"
+    And the exit status should be 0
```

Comment:
I think these last 2 lines could be shortened to:

```
Then it should pass with "0 examples, 0 failures"
```

IMO, that's better for the purposes of documentation.